airflow/providers/cncf/kubernetes/utils/pod_launcher.py

```diff
@@ -51,10 +55,23 @@ def should_retry_start_pod(exception: Exception) -> bool:
 class PodStatus:
     """Status of the PODs"""
 
-    PENDING = 'pending'
-    RUNNING = 'running'
-    FAILED = 'failed'
-    SUCCEEDED = 'succeeded'
+    PENDING = 'Pending'
+    RUNNING = 'Running'
+    FAILED = 'Failed'
+    SUCCEEDED = 'Succeeded'
+
+    terminal_states = {FAILED, SUCCEEDED}
+
+
+def container_is_running(event: V1Pod, container_name: str):
```

Comment:
`event` being a `V1Pod` seems odd to me? Should we rename this to `pod`?

Reply:
yeah this was a point of  weirdness in the existing code, where in many places, `event` would be used to as variable name for the thing returned from api which we would expect to be a pod.  do you know why that language was used?

Reply:
Probably here (the client docs use similar naming in their docs):
https://github.com/apache/airflow/blob/e4c849ec20b61743d64410019f7b6163acab8095/airflow/executors/kubernetes_executor.py#L145

However, that is a dict 🤷‍♂️

Reply:
To clarify, event makes sense when watching, otherwise it doesn't.